plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSplitManager.java

```diff
@@ -291,13 +302,7 @@ public String getTableLocation(SchemaTableName table)
         @Override
         public TableSnapshot getSnapshot(SchemaTableName table, ConnectorSession session)
         {
-            throw new UnsupportedOperationException("Unimplemented");
-        }
-
-        @Override
-        public List<AddFileEntry> getValidDataFiles(SchemaTableName table, ConnectorSession session)
-        {
-            return validDataFiles;
+            return null; // hack, unused
```

Comment:
Or do you need it to return something?
```
throw new UnsupportedOperationException("Unimplemented");
```

Reply:
yes, this method is called, and i cannot throw. it was a lazy choice, since it's confined to this test class